### PR TITLE
Rename labels from sigs.k8s.io to machine.openshift.io

### PR DESCRIPTION
**Pull request by spangenberg**
/cc bison
For reference:
https://github.com/openshift/installer/pull/1263
https://github.com/openshift/cluster-api-provider-aws/pull/161

Also includes change in `tests/integrations/generate.sh` to compliment https://github.com/openshift/cluster-api-provider-aws/pull/164

### CODE REVIEW

File: pkg/controller/machinehealthcheck/machinehealthcheck_controller.go [hunk 1]
```diff
@@ -269,8 +269,8 @@ func hasMatchingLabels(machineHealthCheck *healthcheckingv1alpha1.MachineHealthC
 
 func isMaster(machine mapiv1.Machine, client client.Client) bool {
 	machineMasterLabels := []string{
```
Comment (enxebre) [inline pkg/controller/machinehealthcheck/machinehealthcheck_controller.go]: this will need to be backward compatible for not to break

**Comment by spangenberg** [comment]
> How come test/e2e/operator_expectations.go was deleted? rebase?

Yep, messed up the rebase, pushed again. It's now in cluster-api-actuator-pkg

File: test/integration/generate.sh [hunk 2]
```diff
@@ -10,6 +10,6 @@ metadata:
   namespace: openshift-machine-api
 type: Opaque
 data:
-  awsAccessKeyId: $(echo -n $(aws configure get aws_access_key_id) | base64)
-  awsSecretAccessKey: $(echo -n $(aws configure get aws_secret_access_key) | base64)
+  aws_access_key_id: $(echo -n $(aws configure get aws_access_key_id) | base64)
+  aws_secret_access_key: $(echo -n $(aws configure get aws_secret_access_key) | base64)
```
Comment (enxebre) [inline test/integration/generate.sh]: can you reference the counterpart PR why this i needed?

Comment (spangenberg) [reply test/integration/generate.sh]: https://github.com/openshift/cluster-api-provider-aws/pull/164

**Pull request merged**